data/gamerules.txt

```diff
@@ -45,7 +45,43 @@ gamerules
 	# ALLOWABLE VALUES: any integer >= 0
 	"npc max mining time" 3600
 	
-	# Ships with the "frugal" personality will only use weapons that consume ammo or fuel while their health is at or below this fraction.
+	# Ships with the "frugal" personality will only use weapons that consume ammo or fuel while their
+	# health is at or below this fraction.
 	# DEFAULT: .75 (75 percent)
 	# ALLOWABLE VALUES: any value between 0. and 1.
-	"universal frugal threshold" .75
+	"universal frugal threshold" 0.75
+
+	# The lowest proportion of it's full price the value of a ship or outfit can depreciate to.
+	# By default, a fully depreciated outfit normally worth 1000 credits will sell for 250 credits.
+	# DEFAULT: 0.25
+	# ALLOWABLE VALUES: any value between 0. and 1.
+	"depreciation min" 0.25
+
+	# Number of days before a new outfit and ship begin to depreciate.
+	# DEFAULT: 7 (7 days)
+	# ALLOWABLE VALUES: any integer >= 0.
+	"depreciation grace period" 7
+
+	# First factor of depreciation calculation. The second factor is "depreciation max age" below
+	# This is the proportion of a ship or outfit's value that remains per day after the grace period.
+	# The value is then multiplied by previous value on the next day, and repeat.
+	# So this can be calculated by "remaining daily proportion" = "daily proportion" ^ (age - "grace period")
+	# where age is in days. "remaining daily percent" is then multiplied by result from next factor.
```

Comment:
I'm not sure this is very useful. Some of it seems actively harmful.
`This is the proportion of a ship or outfit's value that remains per day after the grace period.`
This is just not true. It is almost true, and it gets closer to being true as max age increases, but it is never actually correct. Describing it this way, along with what comes next, seems like it will, at best, confuse readers enough that they will have to go figure it out for themselves, or, I think more likely, leave them with a misconception about how depreciation actually works.
```suggestion
```